<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetInfo</class>
 <widget class="QWidget" name="WidgetInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelNumBola">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bola Número </string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonNext">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Ball</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonPrevious">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Ball</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameChart">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>471</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hijos En árbol</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chBNoCerrar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No Cerrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
